reference/viewpoint.md

```diff
@@ -39,11 +40,13 @@ The `near` and the `fieldOfView` fields define together the viewing frustum. Any
 (standing between the camera and the near plane) won't appear.
 
 The `follow` field can be used to specify the name of a robot (or other object)
-that the viewpoint needs to follow during the simulation. If the string is
+that the viewpoint will follow in translation (travelling movement) during the simulation. If the string is
```

Comment:
It seems that travelling with two ``l`` is used especially in British English whether in American English the version with a single ``l`` is more usual.